### ISSUE TITLE
No Mudule named "Utils"

### ISSUE THREAD

**Issue by RezVick** [TheLowSpecPC/MyRepo#7]
<module>
    from Utils import Logger
ModuleNotFoundError: No module named 'Utils'

**Comment by TheLowSpecPC** [comment]
I don't remember using utils in in my project but anyway open cmd and type "pip install utils". This might solve your problem. Come back if there are more issues.

**Comment by RezVick** [comment]
It did not solve it it still give me this:\AppData\Local\Programs\Python\Python311\Lib\site-packages\RedDownloader\RedDownloader.py", line 19, in <module>
    from Utils import Logger
ModuleNotFoundError: No module named 'Utils'

**Comment by TheLowSpecPC** [comment]
I looked into the issue and found out that RedDownloader packge has been updated and its not working properly. RedDownloader calls a pythgon file named 'Utils.py' but its not working for some reason. I have experienced several problems from this packge and I know its infuriating. Only thing we can do for now is to wait until the devs of RedDownloader patch this bug. I will try to make this situation better. 

I will recommend going back to older version of RedDownloader by uninstalling it by typing "pip uninstall RedDownloader" then type "pip install RedDownloader==3.3.0". 

I hope this helps.
